packages/nodes-base/nodes/Aws/AwsSns.node.ts

```diff
@@ -159,9 +185,24 @@ export class AwsSns implements INodeType {
 				displayOptions: {
 					show: {
 						operation: ['publish', 'delete'],
+						target: ['topic'],
```

Comment:
I'm not 100% sure, how to set this logic. 
I want the field to display if:
- operation == 'delete'
- operation == 'publish' && 'target == 'topic' 

How can this be done? Thanks!